docs/fastdds/dds_layer/core/waitsets/waitsets.rst

```diff
@@ -0,0 +1,83 @@
+.. include:: ../../../../03-exports/aliases.include
+.. include:: ../../../../03-exports/aliases-api.include
+
+.. _dds_layer_core_waitsets:
+
+Conditions and Wait-sets
+========================
+
+Conditions (in conjunction with wait-sets) provide an alternative mechanism to allow the middleware
+to communicate communication status changes (including arrival of data) to the application.
```

Comment:
```suggestion
to notify communication status changes (including arrival of data) to the application.
```
To avoid the cacophony of repeating 'communicate/ion'